### PR TITLE
Main Packages should not access Test Packages for Java

### PR DESCRIPTION
**Pull request by scottizu**

### CODE REVIEW

File: examples/java/src/main/java/report/JSONToHTMLConverter.java [hunk 1]
```diff
@@ -15,18 +15,17 @@
...
 import java.util.ArrayList;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
-
 import org.apache.commons.lang3.StringEscapeUtils;
 import org.json.JSONArray;
 import org.json.JSONObject;
-
+import util.HTMLGenerator;
```
Comment (seangates) [inline examples/java/src/main/java/report/JSONToHTMLConverter.java:28]: But ... you removed `package util;`?

**Comment by scottizu** [comment]
Package util still exists.  Util classes are standalone generic classes that can potentially be used in other projects.  This class was moved to a report package since it is not a util.

**Comment by seangates** [comment]
Okay. As long as that was intentional. I'm not a Java expert. 😉

**Comment by scottizu** [comment]
Yes, this was interesting in that the code was actually compiling from Eclipse (IDE), but not Maven (Package Manager).  After the change, code now compiles after download, from both Maven and Eclipse, which is what we want.

**Pull request merged**